docs/collecting-data/collecting-from-own-applications/java-tracker/tracking-specific-client-side-properties/index.md

```diff
@@ -6,122 +6,119 @@ sidebar_position: 40
-| colorDepth       | cd                 | br\_colordepth                        |
-| networkUserId\*  | tnuid              | network\_userid                       |
-| domainUserId     | duid               | domain\_userid                        |
-| domainSessionId  | sid                | domain\_sessionid                     |
+| Property         | Field in raw event | Column(s) in enriched event         |
+|------------------|--------------------|-------------------------------------|
+| userId           | uid                | user_id                             |
+| ipAddress*       | ip                 | user_ipaddress                      |
+| timezone**       | tz                 | os_timezone                         |
+| language         | lang               | br_lang                             |
+| useragent*       | ua                 | useragent                           |
+| viewport         | vp                 | br_viewheight, br_viewwidth         |
+| screenResolution | res                | dvce_screenheight, dvce_screenwidth |
+| colorDepth       | cd                 | br_colordepth                       |
+| networkUserId*   | tnuid              | network_userid                      |
+| domainUserId     | duid               | domain_userid                       |
+| domainSessionId  | sid                | domain_sessionid                    |
 
 Note: the ability to set `domainSessionId` was added in version 0.11.
 
 These properties are especially useful for client-side tracking, or for linking client-side and server-side tracking. We highly recommend tracking in both client-side and server-side, if it's relevant to your application. Read more about this in these [blog](https://snowplowanalytics.com/blog/2019/02/05/how-server-side-tracking-fills-holes-in-your-data-and-improves-your-analytics/) [posts](https://snowplowanalytics.com/blog/2021/11/09/the-unrivaled-power-of-joining-client-and-server-side-tracking/).
 
-If you are also using the Javascript tracker, it will set [cookies](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/cookies-and-local-storage/index.md) in the browser. The `Subject` properties `domainUserId`, and `domainSessionId` are intended to be used for extracted cookie values passed to the Java tracker. If you want to track other identification tokens, we recommend creating schemas and using context entities.
+If you are also using the Javascript tracker, it will set [cookies](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/cookies-and-local-storage/) in the browser. The `Subject` properties `domainUserId`, and `domainSessionId` are intended to be used for extracted cookie values passed to the Java tracker. If you want to track other identification tokens, we recommend creating schemas and using context entities.
```

Comment:
Here it seems that the links were changed from the relative to absolute with the old domain name. We should prefer to use the relative links to the actual `.md` files so that broken links are checked during build.